contracts/middle-layer/resource-mirror/BucketHub.sol

```diff
@@ -127,6 +127,38 @@ contract BucketHub is BucketStorage, GnfdAccessControl, CmnHub, IBucketHub {
         delegateAdditional();
     }
 
+    function prepareCreateBucket(
+        address,
+        CreateBucketSynPackage memory
+    ) external payable onlyMultiMessage returns (uint8, bytes memory, uint256, uint256, address) {
```

Comment:
Do we really need `onlyMultiMessage`modify here? Because the undelying call will check that, seems duplicated to me.

Reply:
no need , removed